--- ..\6\external\webL10n\l10n.js	2017-03-08 13:29:52.000000000 +0200
+++ ..\5\external\webL10n\l10n.js	2017-03-09 20:53:45.072662100 +0200
@@ -827,7 +827,12 @@
   function getL10nData(key, args, fallback) {
     var data = gL10nData[key];
     if (!data) {
-      console.warn('#' + key + ' is undefined.');
+      ///console.warn('#' + key + ' is undefined.'); /// patched
+      // === patch start ===
+      if (Object.keys(gL10nData).length > 0) {
+        console.warn('#' + key + ' is undefined.');
+      }
+      // === patch end === 
       if (!fallback) {
         return null;
       }
