--- overlay_manager_old.js	2017-03-08 13:29:52.000000000 +0200
+++ overlay_manager.js	2017-03-09 20:55:28.198062100 +0200
@@ -49,7 +49,14 @@
       if (!name || !element || !(container = element.parentNode)) {
         throw new Error('Not enough parameters.');
       } else if (this.overlays[name]) {
-        throw new Error('The overlay is already registered.');
+        ///throw new Error('The overlay is already registered.'); /// patched
+        // === patch start ===
+        if (this.active !== name) {
+            this.unregister(name);
+          } else {
+            throw new Error('The overlay is already registered and active.');
+          }    
+        // === patch start === 
       }
       this.overlays[name] = { element: element,
                               container: container,
