--- viewer_old.js	2017-03-09 21:03:17.354262100 +0200
+++ viewer.js	2017-03-08 13:29:52.000000000 +0200
@@ -188,12 +188,5 @@
     document.readyState === 'complete') {
   webViewerLoad();
 } else {
-  ///document.addEventListener('DOMContentLoaded', webViewerLoad, true); /// patched
-  // === patch start ===
-  PDFJS.webViewerLoad = function (src) {
-    if (src) DEFAULT_URL = src;
- 
-    webViewerLoad();
-  }  
-  // === patch end === 
+  document.addEventListener('DOMContentLoaded', webViewerLoad, true);
 }
